src/app/components/common/courthouse/courthouse.component.ts

```diff
@@ -48,6 +49,7 @@ export class CourthouseComponent implements AfterViewInit, OnChanges {
     if (this.courthouses.length) {
       this.props.element = this.autocompleteContainer.nativeElement as HTMLElement;
       this.props.source = this.courthouses.map((ch) => ch.courthouse_name);
+      this.props.defaultValue = this.courthouse;
```

Comment:
Can you add a unit test for this?

Reply:
Yes. I need to take a look at the fallback too actually. In the event of no courthouses returned.